### ISSUE TITLE
Support Kotlin code

### ISSUE THREAD

**Issue by noproxy** [MegatronKing/StringFog#28]
After using StringFog Plugin, all Kotlin code get loss when transform.

Consider:
1. copy as it is for all Kotlin class to make this plugin work for project which has kotlin code
2. support obfuscate string in Kotlin code

**Comment by noproxy** [comment]
String Fog keeps deleting outputs if ``dirInput``'s incremental = false.
```groovy
   if (!dirInputs.isEmpty() || !jarInputs.isEmpty()) {
            File dirOutput = transformInvocation.outputProvider.getContentLocation(
                    "classes", getOutputTypes(), getScopes(), Format.DIRECTORY)
            FileUtils.mkdirs(dirOutput)
            if (!dirInputs.isEmpty()) {
                dirInputs.each { dirInput ->
                    if (transformInvocation.incremental) {
                        ...
                    } else {
                        if (dirOutput.exists()) {
                            // some outputs get lost here !!
                            dirOutput.deleteDir()
                        }
                    }
             }
    }
```

That's ok if there is only one dirInput. But when it's coming to kotlin, there will be two ``dirInput``. So some class file get lost.

**Comment by noproxy** [comment]
close because code merged in #26

**Comment by MegatronKing** [comment]
Updated plugin version 2.1.0
